numcodecs/checksum32.py

```diff
@@ -168,3 +152,20 @@ def decode(self, buf, out=None):
             out.view("uint8")[:] = b[:-4]
             return out
         return memoryview(b[:-4])
+
+
+if _crc32c:
+
+    class CRC32C(Checksum32):
+        """Codec add a crc32c checksum to the buffer.
+
+        Parameters
+        ----------
+        location : 'start' or 'end'
+            Where to place the checksum in the buffer.
+        """
+
+        codec_id = 'crc32c'
+
+        checksum = _crc32c.crc32c
```

Comment:
So this works when simply installing `crc32c` and doing `import crc32c as _crc32c` as above

However mypy seems to [dislike this]( https://results.pre-commit.ci/run/github/68207196/1731548166.F8fD3_m4TDKH1jxKLw6gsA )

```python
numcodecs/checksum32.py:170: error: Item "None" of Module | None has no attribute "crc32c"  [union-attr]
```

Reply:
Added comments for mypy to ignore the lines it was concerned about. Seems to have addressed the issue. These are not actual problems after all. Just mypy being picky